### DISCUSSION TITLE
in githubuniverse discussions, New Discussion button is missing sometimes

### DISCUSSION THREAD

**Discussion by louking** [community/community#2915] [Discussions] [answered]
I was finally able to make a discussion post, but now I don't see the New Discussions button -- weird transient problem. But New Discussions was (obviously) available in this repo.

ctrl-F5 cache buster doesn't resolve it.

https://github.com/githubevents/universe2020/discussions

**Comment by becca** [comment, marked as answer]
Yes, we disabled the ability to create new discussions when the event is not live broadcasting. The second pinned discussion explains this https://github.com/githubevents/universe2020/discussions/353

**Reply by louking** [reply]
Oh, oops. Didn't see that. Thanks for the explanation

**Reply by louking** [reply]
actually where/how is it pinned? I only see the most recent discussions

![image](https://user-images.githubusercontent.com/875138/101695412-85aea580-3a42-11eb-9cc6-e78a1a9e749c.png)

**Reply by becca** [reply]
it's the large blue discussion at the top of the list

**Reply by louking** [reply]
Oh! Hmm. Not how I would have coded it but you guys are the UI experts. My eyes just skipped over the two big boxes on the top.
